### ISSUE TITLE
Get data from Smart filter select field to Jet Engine Form select field.

### ISSUE THREAD

**Issue by KaloyanDDimitrov** [CrocoBlock/suggestions#6812]
Hello,
We are trying to create a Rent a car website. One of the problems we have is we cant get the data from the smart selector field and send it to the form. The idea is when the client enters the site there is a filter form to filter the cars by pick up location return location and check in/out date. When the client selects a car in the single car page there is a form for addons, location , rent date and input fields for name , email , phone and message. We want to auto populate the input select fields for location and date with the help of the smart filters, so we dont have to make the client select the location again.
The second problem is we cant make an addon like insurance auto selected(checked) by default.
![344300543_1894266774267845_2835717613647974635_n](https://github.com/Crocoblock/suggestions/assets/39025645/e80842ba-19b3-4531-941a-1fed960ad2d3)
![344111159_183730231247606_1000273426606267139_n](https://github.com/Crocoblock/suggestions/assets/39025645/23847764-d917-4f00-a480-6c89a08b16ec)

**Comment by videovisioI** [comment]
Hi there, did u solve it'?  we are trying to pass the same information "checkincheckout" field from the first step jetsmartfilter to the jetform . thanks

**Comment by KaloyanDDimitrov** [comment]
> Hi there, did u solve it'? we are trying to pass the same information "checkincheckout" field from the first step jetsmartfilter to the jetform . thanks

Yes we did. Using Js and saving selected id in session then importing it in the form. Is not the most elegant solution but it works. Now we have a problem with limiting the minimum days in the date filter.
